### PR TITLE
Update project to prepare for new direction: SJ++

### LINKED ISSUE #3: Update project `README`
- Add UI mockup (`docs/images/Ui.png`)
- Update the link of the GitHub Actions build status badge
- Acknowledge the original source of the code e.g., This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
- Update site-wide settings as described in the guide [Using Jekyll for project documentation @SE-EDU/guides](https://se-education.org/guides/tutorials/jekyll.html).
 The text AB-3 in `docs\_sass\minima\_base.scss` needs updating too.

### PR DESCRIPTION
**Pull request by wxwern**
This PR changes AddressBook references in the homepage to SJ++ and `.gitignore` updates to prepare for our project work.

**Note:** UG and DG are not yet modified.

Closes #3.

### CODE REVIEW

**Review by CJ-Lee01 — APPROVED** [review]
So far most of the changes seems to be changing the AB-3 description to our project description. I agree with the changes made. Looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+## SteveJobs++ (SJ++)
+
+[![CI Status](https://github.com/AY2324S1-CS2103T-W08-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S1-CS2103T-W08-3/tp/actions)
+
+SJ++ is a desktop application helps manage job applications by allowing you to easily record organization and recruiter contacts, plus note down information about your applications. It is optimized for CLI use, so you can operate the application with only your keyboard.
```
Comment (McNaBry) [inline README.md:5]: Minor phrasing but should be "... desktop application **that** helps **to** manage..."

**Review by McNaBry — CHANGES_REQUESTED** [review]
Overall, LGTM

File: README.md (hunk 1 above)
Comment (wxwern) [inline README.md:5]:
```suggestion
SJ++ is a desktop application that helps you manage your job applications by easily recording organization and recruiter contacts, plus note down information about your applications. It is optimized for CLI use, so you can operate the application with only your keyboard.
```

File: .gitignore [hunk 2]
```diff
@@ -15,9 +15,15 @@ src/main/resources/docs/
...
 
+# IDE config files
+.project
+.settings/
+
 # Test sandbox files
 src/test/data/sandbox/
 
-# MacOS custom attributes files created by Finder
+# Attributes files created by Finder and File Explorer
```
Comment (wxwern) [inline .gitignore:25]:
```suggestion
# Attribute files created by Finder and File Explorer
```

**Review by McNaBry — APPROVED** [review]
Ok, the previous phrasing error has been corrected. Overall, the changes looks good to me.

**Review by tanshiyu1999 — COMMENTED** [review]
LGTM

**Review by tanshiyu1999 — APPROVED** [review]
LGTM

**Pull request merged**